modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -0,0 +1,38 @@
+= API Development in Studio 7.4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To ensure that your Mule applications always refer to a single source of truth for your API specifications, MuleSoft introduces a workflow to design, publish, and implement your API specifications.
+
+. Design an API specification in Design Center. +
+You can either xref:design-center::design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[Create and Publish an API Specification with the Code Editor].
+. Publish that API specification to Exchange. +
+Make your newly designed API available to your entire organization as an Exchange dependency. As you change your API, you can publish new versions of your API specifications to Exchange. See xref:exchange::to-change-raml-version.adoc[best practices for API versioning].
+. Import the API specification from Exchange to Studio. +
+Your API specification is imported to Anypoint Studio as a dependency. This means that your Mule application project now holds a pointer to the specific API version that you are implementing. Studio monitors your API and notifies you when there is a new version available. See xref:api-sync.adoc[API Sync] for how to update to new versions of your API specifications in Studio.
+
+This workflow allows you to use both RAML and OAS specifications.
+
+== Exchange as a Centralized Repository for All Your Project Dependencies
```

Comment:
It is not completely clear if local copies were only available in previous versions of Studio, and now only API references are used, or now both options can be selected when importing an API in Studio 7.4 (copy and reference)